.github/workflows/node-test.yml

```
@@ -21,8 +21,8 @@ jobs:
 
             - name: Lint
               run: |
-                  yarn d2-style js check
-                  yarn d2-style text check
+                  yarn d2-style check js
+                  yarn d2-style check text
```

With 8.x you can do this to run all `check`s with a single command:

```suggestion
                  yarn d2-style check
```